pyproject.toml

```diff
@@ -39,14 +40,21 @@ select = [
     "PYI032", # use `object`, not `Any`, as the second parameter to `__eq__`
     "PYI055", # multiple `type[T]` usages in a union
     "PYI058", # use `Iterator` as the return type for `__iter__` methods
-    "UP004", # Remove explicit `object` inheritance
-    "UP006", # PEP-585 autofixes
-    "UP007", # PEP-604 autofixes
-    "UP013", # Class-based syntax for TypedDicts
-    "UP014", # Class-based syntax for NamedTuples
-    "UP019", # Use str over typing.Text
-    "UP035", # import from typing, not typing_extensions, wherever possible
-    "UP039", # don't use parens after a class definition with no bases
+]
+ignore = [
+    # Slower and more verbose https://github.com/astral-sh/ruff/issues/7871
+    "UP038", # Use `X | Y` in `isinstance` call instead of `(X, Y)`
+]
+
+[tool.ruff.lint.per-file-ignores]
+# Disable "mordernization" rules with autofixes from test cases as they often
```

Comment:
```suggestion
# Disable "modernization" rules with autofixes from test cases as they often
```

Reply:
I think I'm seriously starting to consider in-editor spellchecking